finetune/base_models/gpt2/encoder.py

```diff
@@ -51,6 +51,21 @@ class GPT2Encoder(BaseEncoder):
 
     def __init__(self, encoder_path=ENCODER_PATH, vocab_path=VOCAB_PATH):
         super().__init__(encoder_path=encoder_path, vocab_path=vocab_path)
+        self.freqs = {}
+        index = 0
+        with open("dict.txt", "r", encoding="utf-8") as freq_dict:
+            lines = freq_dict.readlines()
+            for line in lines:
+                idx = line.rfind(" ")
+                if idx == -1:
+                    raise ValueError(
+                        "Incorrect dictionary format, expected '<token> <cnt>'"
+                    )
+                if "madeupword" in line[:idx]:
```

Comment:
Seems like this might have been for testing?

Reply:
Maybe... but for Facebook's testing purposes :P

Reply:
No facebook literally has that in their files